docs/spec/circuit-breaker/01_concepts.md

```diff
@@ -0,0 +1,16 @@
+# Concepts
+
+The intention of the circuit breaker is to have a contingency plan for a
+running network which maintains network liveness. This can be achieved through
+selectively "pausing" functionality of specific modules on a running network.
+The circuit breaker is intended to be enabled through either:
+ - governance, 
```

Comment:
```suggestion

 - governance, 
```

Reply:
In what way - certainly a different process than for regular proposals, right?

Reply:
well I don't see why governance should not also be able to trigger a circuit breaker in non emergency situations